papers/helge_eichhorn/eichhorn.rst

```diff
@@ -0,0 +1,318 @@
+:author: Helge Eichhorn
+:email: eichhorn@dik.tu-darmstadt.de
+:institution: Technische Universität Darmstadt, Department of Computer Integrated Design
+
+:author: Reiner Anderl
+:email: anderl@dik.tu-darmstadt.de
+:institution: Technische Universität Darmstadt, Department of Computer Integrated Design
+
+--------------------------------------------------
+Plyades: A Python Library for Space Mission Design
+--------------------------------------------------
+
+.. class:: abstract
+
+    Designing a space mission is a computation-heavy task.
+    Software tools that conduct the necessary numerical simulations and optimizations are therefore indispensable.
+    The usability of existing software, written in Fortran and MATLAB, suffers because of high complexity, low levels of abstraction and out-dated programming practices.
+    We propose Python as a viable alternative for astrodynamics tools and demonstrate the proof-of-concept library Plyades which combines powerful features with Pythonic ease of use.
+
+.. class:: keywords
+
+   data modeling, object-oriented programming, orbital mechanics, astrodynamics
+
+Introduction
+------------
+
+Designing a space mission trajectory is a computation-heavy task.
+Software tools that conduct the necessary numerical simulations and optimizations are therefore indispensable and high numerical performance is required.
+Since no mission or spacecraft is alike the ever-changing requirements and constraints also demand high development speed and programmer productivity.
+Due to its supreme numerical performance Fortran has been the top programming language in many organizations within the astrodynamics community.
+
+At the European Space Operations Center (ESOC) of the European Space Agency (ESA) a large body of sometimes decades old Fortran77 code remains in use for mission analysis tasks.
+While this legacy code mostly fulfills the performance requirements [#]_ usability and programmer productivity suffer.
+One reason for this is that Fortran is a compiled language which hinders exploratory analyses and rapid prototyping.
+The low level of abstraction supported by Fortran77 and programming conventions, like 6-character variable name limits and fixed-form source code, that are in conflict with today's best practices are a more serious problem, though.
```

Comment:
Almost fixed ;-)

I like "the 6-character variable name limit and ~~the~~ fixed-form source code" better.